### ISSUE TITLE
parse error when using colon in python 

### ISSUE THREAD

**Issue by kaizeng** [KxSystems/jupyterq#20]
It looks like JupyterQ doesn't like ":" in python codes: 

````
/%python
d={"a":2, "b":3, "c":4}
````

the error I get:

````
parse error
assign
````

Is there a workaround?

Thank you very much for your help!

**Comment by jhanna-kx** [comment]
Hello,
This should work in version 1.1, can you post the output from the Help->About option in the Notebook?
Thanks

**Comment by kaizeng** [comment]
I am using an old version
````
KDB+ v3.5 2017.07.19 kdb+ kernel for jupyter v 0.0.1
````
let me try the latest version and report back. Thanks!

**Comment by kaizeng** [comment]
yes, I confirm that the issue is fixed in 1.1. Thank you very much for your help!
